test/index.test.js

```diff
@@ -108,6 +108,8 @@ describe('Index Tests', async () => {
       Vary2: 'X-Token, X-Service',
       'Cache-Control': 'max-age=300',
     });
+    assert.deepEqual(result.body.requestParams, { limit: 3 });
+    assert.equal(result.body.desc, 'some fake comments that mean nothing');
```

Comment:
I think `description` would be a more *descriptive* name than just `desc`. The additional characters won't hurt us.